alf/layers.py

```diff
@@ -29,6 +31,40 @@ def identity(x):
     return x
 
 
+def square(x):
+    """torch doesn't have square."""
+    return torch.pow(x, 2)
+
+
+def batch_normalization(x, mean, variance, variance_epsilon):
```

Comment:
It's indeed literally "batch normalization"

Reply:
I understand this function implements a specific way to normalize a batch of data, but the name ```batch_normalization``` is typically refers to the specific [operation](https://en.wikipedia.org/wiki/Batch_normalization), especially under the context of DNN. Since there are several difference with the standard version, it's better to use a different name, maybe ```batch_ standardization```.

Reply:
> I understand this function implements a specific way to normalize a batch of data, but the name `batch_normalization` typically refers to the specific [operation](https://en.wikipedia.org/wiki/Batch_normalization), especially under the context of DNN. Since there are several difference with the standard version, it's better to use a different name, maybe `batch_ standardization`.

That's what tf2 calls. I can change it to `normalize_along_batch_dims`.